Deploy-Dependencies.ps1

```diff
@@ -9,14 +9,36 @@ function Get-PeviewBinary {
 
   # use temporary folder for download
   New-Item -ItemType Directory -Force -Path "$($env:TEMP)/tmp";
-  Set-Location "$($env:TEMP)/tmp";
+  Set-Location "$($env:TEMP)/tmp"
 
-  &wget $Url -OutFile "processhacker-2.39-bin.zip";
-  $PhArchiveHash = (Get-FileHash -Algorithm SHA256 -Path "./processhacker-2.39-bin.zip").Hash;
+  $apiUrl = 'https://ci.appveyor.com/api';
+  $headers = @{
+    "Content-type" = "application/json"
+  };
+  $accountName = 'processhacker';
+  $projectName = 'processhacker';
+  $artifactFileName = "processhacker-build-bin.zip";
 
-  if ($PhArchiveHash -eq "2afb5303e191dde688c5626c3ee545e32e52f09da3b35b20f5e0d29a418432f5") {
-    &7z.exe x "./processhacker-2.39-bin.zip" "$($env:platform)/peview.exe";
-    $PeviewBinaryFile = (Resolve-Path "./$($env:platform)/peview.exe").Path;
+  # get project with last build details
+  $build = Invoke-RestMethod -Method Get -Uri "$apiUrl/projects/$accountName/$projectName/build/$BuildVersion" -Headers $headers
+
+  # processhacker builds have a single job
+  # get the job id
+  $jobId = $build.build.jobs[0].jobId;
+
+  # get the zip file job artifacts
+  Invoke-WebRequest -Uri "$apiUrl/buildjobs/$jobId/artifacts/$artifactFileName" -OutFile "./$artifactFileName"
+
+  $PhArchiveHash = (Get-FileHash -Algorithm SHA256 -Path "./$artifactFileName").Hash;
+
+  $pePlatform = "32bit"
```

Comment:
This is required because the folders inside the zip changed from x86 and x64 to 32bit and 64bit respectively